_partials/python_checkup.md

```diff
@@ -15,4 +15,12 @@ Now run the following command to check if you can load these packages:
 python -c "$(curl -fsSL <PIP_LOADER_URL>)"
 ```
 
+Make sure you can run Jupyter:
+
+```bash
+jupyter notebook
+```
+
+And open a `Python 3` notebook.
```

Comment:
```suggestion
And open a `Python 3` notebook.

Make sure that you are running the correct python version in the notebook. Open a cell and run :
``` python
import sys; sys.version
``` .
```

👆 no idea how to add code blocs properly within suggestions 😅

Reply:
lgtm